src/main/java/oshi/software/os/mac/local/CentralProcessor.java

```diff
@@ -185,9 +192,35 @@ public void setFamily(String _family) {
 	 * {@inheritDoc}
 	 */
 	public float getLoad() {
-		ArrayList<String> topResult = ExecutingCommand.runNative("top -l 1 -R -F -n1"); // cpu load is in [3]
-		String[] idle = topResult.get(3).split(" "); // idle value is in [6]
-		return 100 - Float.valueOf(idle[6].replace("%", ""));
+		int machPort = SystemB.INSTANCE.mach_host_self();
+
+		int[] prevTicks = new int[SystemB.CPU_STATE_MAX];
+		HOST_CPU_LOAD_INFO cpuLoadInfo = new HOST_CPU_LOAD_INFO();
+		int ret = SystemB.INSTANCE.host_statistics(machPort,
+				SystemB.HOST_CPU_LOAD_INFO, cpuLoadInfo, new IntByReference(
+						SystemB.HOST_CPU_LOAD_INFO_COUNT));
+		if (ret == 0) {
+			for (int i = 0; i < SystemB.CPU_STATE_MAX; i++) {
+				prevTicks[i] = cpuLoadInfo.cpu_ticks[i];
+			}
+		}
+		try {
+			Thread.sleep(1000);
+		} catch (InterruptedException e) {
+		}

```

Comment:
I could display load factor / nCPUs for a decent approximation without waiting 1000 milliseconds to count cpu ticks...